moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -123,8 +123,10 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
   {
     const robot_state::RobotState& values = moveit::planning_interface::MoveGroupInterface::getJointValueTarget();
     bp::list l;
-    for (const double *it = values.getVariablePositions(), *end = it + getVariableCount(); it != end; ++it)
```

Comment:
I think, it will suffice to call `values.getVariableCount()` to get the number overall number of variables in the model (in contrast to `getVariableCount()` that returns the variable count of the joint model group...)